handlers/common.go

```diff
@@ -199,7 +200,9 @@ func replyWithParent(replyData map[string]interface{}, event *revents.Event, par
 	if len(replyData) > 0 {
 		empty = "not empty"
 	}
-	logrus.Infof("Reply: %v, %v, %v:%v, data: %v", parentUUID, reply.Name, reply.ResourceId, reply.ResourceType, empty)
+	if event.Name != "ping" {
```

Comment:
I think we have duplicate PR for suppress ping logs https://github.com/rancher/agent/pull/50